src/toil_scripts/adam_uberscript/adam_uberscript.py

```diff
@@ -0,0 +1,408 @@
+
+scaling_initial_wait_period_in_seconds = 300
+cluster_scaling_interval_in_seconds = 300
+
+cluster_size_lock = threading.Lock()
+
+aws_region = 'us-west-2'
+
+ref = "https://s3-us-west-2.amazonaws.com/cgl-pipeline-inputs/variant_grch38/GRCh38_full_analysis_set_plus_decoy_hla.fa"
+amb = "https://s3-us-west-2.amazonaws.com/cgl-pipeline-inputs/variant_grch38/GRCh38_full_analysis_set_plus_decoy_hla.fa.amb"
+ann = "https://s3-us-west-2.amazonaws.com/cgl-pipeline-inputs/variant_grch38/GRCh38_full_analysis_set_plus_decoy_hla.fa.ann"
+bwt = "https://s3-us-west-2.amazonaws.com/cgl-pipeline-inputs/variant_grch38/GRCh38_full_analysis_set_plus_decoy_hla.fa.bwt"
+pac = "https://s3-us-west-2.amazonaws.com/cgl-pipeline-inputs/variant_grch38/GRCh38_full_analysis_set_plus_decoy_hla.fa.pac"
+sa = "https://s3-us-west-2.amazonaws.com/cgl-pipeline-inputs/variant_grch38/GRCh38_full_analysis_set_plus_decoy_hla.fa.sa"
+fai = "https://s3-us-west-2.amazonaws.com/cgl-pipeline-inputs/variant_grch38/GRCh38_full_analysis_set_plus_decoy_hla.fa.fai"
+alt = "https://s3-us-west-2.amazonaws.com/cgl-pipeline-inputs/variant_grch38/GRCh38_full_analysis_set_plus_decoy_hla.fa.alt"
+phase = "https://s3-us-west-2.amazonaws.com/cgl-pipeline-inputs/variant_grch38/ALL.wgs.1000G_phase3.GRCh38.ncbi_remapper.20150424.shapeit2_indels.vcf.gz"
+mills = "https://s3-us-west-2.amazonaws.com/cgl-pipeline-inputs/variant_grch38/Mills_and_1000G_gold_standard.indels.b38.primary_assembly.vcf.gz"
+dbsnp = "https://s3-us-west-2.amazonaws.com/cgl-pipeline-inputs/variant_grch38/ALL_20141222.dbSNP142_human_GRCh38.snps.vcf.gz"
+omni = "https://s3-us-west-2.amazonaws.com/cgl-pipeline-inputs/variant_grch38/ALL.wgs.1000G_phase3.GRCh38.ncbi_remapper.20150424.shapeit2_indels.vcf.gz"
+hapmap = "https://s3-us-west-2.amazonaws.com/cgl-pipeline-inputs/variant_grch38/ALL_20141222.dbSNP142_human_GRCh38.snps.vcf.gz"
+
+
+def launch_cluster(params):
+    """
+    Launches a toil cluster of size N, with shared dir S, of instance type I, at a spot bid of B
+
+    params: argparse.Namespace      Input arguments
+    """
+    log.info('Launching cluster of size: {} and type: {}'.format(params.num_workers, params.instance_type))
+    subprocess.check_call(['cgcloud',
+                           'create-cluster',
+                           '--leader-instance-type', params.leader_type,
+                           '--instance-type', params.instance_type,
+                           '--share', params.share,
+                           '--num-workers', str(params.num_workers),
+                           '--cluster-name', params.cluster_name,
+                           '--spot-bid', str(params.spot_bid),
+                           '--leader-on-demand',
+                           '--num-threads', str(params.num_workers),
+                           '--ssh-opts',
+                           '-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no',
+                           'toil'])
+
+
+def place_boto_on_leader(params):
```

Comment:
Don't need to fix this for the recompute, but after we're done, let's allow the user to supply a `.aws/credentials` file as well. I've opened #115 to track this.